app/models/host_info_providers/static_info.rb

```diff
@@ -0,0 +1,66 @@
+module HostInfoProviders
+  class StaticInfo < HostInfo::Provider
+    def host_info
+      # Static parameters
+      param = {}
+
+      param["hostgroup"] = host.hostgroup.to_label unless host.hostgroup.blank?
+      param["comment"] = host.comment unless host.comment.blank?
+
+      add_network_params param
+      add_taxonomy_params param
+      add_domain_params param
+      add_login_params param
+      add_unattended_params param
+
+      # Parse ERB values contained in the parameters
+      param = SafeRender.new(:variables => { :host => self }).parse(param)
+
+      { 'parameters' => param }
+    end
+
+    private
+
+    def add_unattended_params(param)
+      return unless SETTINGS[:unattended]
+
+      param["root_pw"] = host.root_pass unless (!host.operatingsystem.blank? && host.operatingsystem.password_hash == 'Base64')
+    end
+
+    def add_login_params(param)
+      owner = host.owner
+      return unless SETTINGS[:login] && owner
+
+      param["owner_name"]  = owner.name
+      param["owner_email"] = owner.is_a?(User) ? owner.mail : owner.users.map(&:mail)
```

Comment:
I believe, this got lost during the rebase:

param["ssh_authorized_keys"] = ssh_authorized_keys
param["foreman_users"] = owner.to_export